go-controller/pkg/util/node_annotations.go

```diff
@@ -428,6 +428,20 @@ func ParseNodeManagementPortMACAddress(node *kapi.Node) (net.HardwareAddr, error
 	return net.ParseMAC(macAddress)
 }
 
+// ParseNodeManagementPortMACAddresses parses the 'OvnNodeManagementPortMacAddresses' annotation
+// for the specified network in 'netName' and returns the mac address.
+func ParseNodeManagementPortMACAddresses(node *kapi.Node, netName string) (net.HardwareAddr, error) {
+	macAddressMap, err := parseNetworkMapAnnotation(node.Annotations, OvnNodeManagementPortMacAddresses)
+	if err != nil {
+		return nil, newAnnotationNotSetError("macAddress annotation not found for node %q ", node.Name)
```

Comment:
the parseNetworkMapAnnotation is already returning  newAnnotationNotSetError I think we should just wrap the error from there, it doing right we can check later on the wrapped error. Also doing this enmasquerade non not set errors.

Reply:
fixed this as well, nice catch.